src/Plugin.php

```diff
@@ -211,6 +218,13 @@ public static function init() {
       }
     }
 
+    // Schedules delete orphan variants cron.
+    if (!wp_next_scheduled(self::CRON_EVENT_ORPHANS_VARIANTS_CLEANUP)) {
+      wp_schedule_event(time(), 'weekly', self::CRON_EVENT_ORPHANS_VARIANTS_CLEANUP);
+    }
+
+    add_action(self::CRON_EVENT_ORPHANS_VARIANTS_CLEANUP, __NAMESPACE__  . '\WooCommerce::cron_orphans_variants_cleanup');
```

Comment:
Let's move this above the if condition to retain consistency with the other cron setup lines above.